components/add_groups_to_team_modal/add_groups_to_team_modal.tsx

```diff
@@ -234,21 +255,22 @@ export default class AddGroupsToTeamModal extends React.PureComponent {
 
         let addError = null;
         if (this.state.addError) {
-            addError = (<div className='has-error col-sm-12'><label className='control-label font-weight--normal'>{this.state.addError}</label></div>);
+            addError = (<div className='has-error col-sm-12'><label
+                className='control-label font-weight--normal'>{this.state.addError}</label></div>);
```

Comment:
please use proper indentation:

```suggestion
            addError = (
                <div className='has-error col-sm-12'>
                    <label className='control-label font-weight--normal'>
                        {this.state.addError}
                    </label>
                </div>
            );
```

Reply:
I think that such rules could be automatically checked by eslint and prettier integration. https://prettier.io/docs/en/related-projects.html#eslint-integrations

Reply:
They can... we are not using prettier though ... I will take this discussion to the team and see if we can implement something like that